content/en/docs/concepts/glossary.md

```diff
@@ -9,189 +9,369 @@ others. This page captures terminology used in the project and what it means.
-  span context and `Baggage` in `Spans`.
-- **[Proto](https://github.com/open-telemetry/opentelemetry-proto):** Language
-  independent interface types.
-- **[Receiver](/docs/collector/configuration/#receivers):** Term used by the
-  `Collector` to define how telemetry data is received. Receivers can be push or
-  pull based.
-- **Request:** See `Distributed Tracing`.
-- **Resource:** Captures information about the entity for which telemetry is
-  recorded. For example, a process producing telemetry that is running in a
-  container on Kubernetes has a pod name, it is in a namespace and possibly is
-  part of a deployment which also has a name. All three of these attributes can
-  be included in the `Resource` and applied to any data source.
-- **[REST](https://en.wikipedia.org/wiki/Representational_state_transfer):**
-  Representation State Transfer.
-- **[RPC](https://en.wikipedia.org/wiki/Remote_procedure_call):** Remote
-  Procedure Call.
-- **[Sampling]({{< relref "/docs/reference/specification/trace/sdk#sampling" >}}):**
-  A mechanism to control the amount of data exported. Most commonly used with
-  the `Tracing` `Data Source`.
-- **SDK:** Software Development Kit. Refers to a telemetry SDK that denotes a
-  `Library` that implement the OpenTelemetry `API`.
-- **Semantic Conventions:** Defines standard names and values of `Metadata` in
-  order to provide vendor-agnostic telemetry data.
-- **Service:** A component of an `Application`. Multiple instances of a
-  `Service` are typically deployed for high availability and scalability. A
-  `Service` may be deployed in multiple locations.
-- **[Span]({{< relref "/docs/reference/specification/trace/api#span" >}}):**
-  Represents a single operation within a `Trace`.
-- **Span Link:** A span link is a link between causally-related spans. For
-  details see [Links between
-  spans]({{< relref "/docs/reference/specification/overview#links-between-spans" >}})
-  and [Specifying
-  Links]({{< relref "/docs/reference/specification/trace/api#specifying-links" >}}).
-- **[Specification](/docs/concepts/components/#specification):** Describes the
-  cross-language requirements and expectations for all implementations.
-- **[Status]({{< relref "/docs/reference/specification/trace/api#set-status" >}}):**
-  The result of the operation. Typically used to indicate whether an error
-  occurred.
-- **Tag:** see `Metadata`.
-- **[Trace]({{< relref "/docs/reference/specification/overview#traces" >}}):** A
-  `DAG` of `Spans`, where the edges between `Spans` are defined as parent/child
-  relationship.
-- **[Tracer]({{< relref "/docs/reference/specification/trace/api#tracer" >}}):**
-  Responsible for creating `Spans`.
-- **Transaction:** See `Distributed Tracing`.
-- **[zPages][]:** An in-process alternative to external exporters. When
-  included, they collect and aggregate tracing and metrics information in the
-  background; this data is served on web pages when requested.
+### **Aggregation**
```

Comment:
Wow, nice update to the glossary. I don't know how long it will take before this PR is merged, but if the delay is too long, I'd pull this out into a separate PR and merge it first. Thoughts @avillela?

Reply:
Yes, this is a fantastic update but I think it might be easiest to pull it out into a separate PR. It would make reviewing much easier.